class_based_auth_views/forms.py

```diff
@@ -0,0 +1,83 @@
+from __future__ import unicode_literals
+
+import warnings
+
+from django import forms
+from django.utils.text import capfirst
+from django.utils.translation import ugettext_lazy as _
+from django.contrib.auth import authenticate, get_user_model
+
+
+class AuthenticationMixin(object):
+    """
+    Base class for authenticating users. Extend this to get a form that accepts
+    username/password logins.
+    """
+
+    error_messages = {
+        'invalid_login': _("Please enter a correct %(username)s and password. "
+                           "Note that both fields may be case-sensitive."),
+        'inactive': _("This account is inactive."),
+    }
+
+    def _init_fields(self):
+        super(AuthenticationMixin, self)._init_fields()
+        self.fields['username'] = forms.CharField(max_length=254)
+        self.fields['password'] = forms.CharField(label=_("Password"), widget=forms.PasswordInput)
+
+    def __init__(self, request=None, *args, **kwargs):
+        """
+        The 'request' parameter is set for custom auth use by subclasses.
+        The form data comes in via the standard 'data' kwarg.
+        """
+        self.request = request
+        self.user_cache = None
+        super(AuthenticationMixin, self).__init__(*args, **kwargs)
+
+        # Set the label for the "username" field.
+        user_model = get_user_model()
```

Comment:
get_user_model je pro is-core použit z jiné knihonvy, jak to řešíš tam, měl bys na to mít metodu ve fomuláři